src/pch.h

```diff
@@ -64,6 +64,9 @@
 #include <rapidjson\istreamwrapper.h>
 #include <rapidjson\ostreamwrapper.h>
 #include <rapidjson\writer.h> // has stringbuffer.h
+
+/* gsl stuff */
+#include <gsl\gsl>
```

Comment:
Maybe they fixed it but one CppCoreGuideline warning is triggered from their `make_not_null `function.